<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agenda | Principal </string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>791</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Contatos</string>
    </property>
    <widget class="QTableWidget" name="tabela_listagem">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>771</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-background-color: rgb(73, 111, 39);
selection-color: rgb(255, 255, 255);</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Cod</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nome</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Telefone</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="bt_editar">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>450</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>EDITAR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_cadastrar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>INSERIR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_excluir">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>450</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>EXCLUIR</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Agenda de Contatos</string>
    </property>
    <widget class="QPushButton" name="bt_sair">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SAIR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_pesquisar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PESQUISAR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="busca">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAgenda">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionNovo_Contato"/>
    <addaction name="actionSair"/>
    <addaction name="separator"/>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menuAgenda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNovo_Contato">
   <property name="text">
    <string>Inserir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>busca</tabstop>
  <tabstop>bt_pesquisar</tabstop>
  <tabstop>bt_sair</tabstop>
  <tabstop>bt_cadastrar</tabstop>
  <tabstop>bt_editar</tabstop>
  <tabstop>bt_excluir</tabstop>
  <tabstop>tabela_listagem</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
